From 003b73552d280d68d42bef21d82fab66157fdf0a Mon Sep 17 00:00:00 2001
From: Alyson Deives Pereira <alyson.deivespereira@windriver.com>
Date: Fri, 16 Dec 2022 16:40:00 -0300
Subject: [PATCH] Change "unable to route event" message log level to debug

After ZeroRPC finishes the handle of the call, it will try to reply OK
event to the caller. In fanout_cast/cast messages in sysinv-conductor
and sysinv-agent, there is no one registered to receive the response,
so it is discarded.

Change this log from warning to debug, since it occurs frequently and
is misleading log analysis in case of real issues.

Signed-off-by: Alyson Deives Pereira <alyson.deivespereira@windriver.com>
---
 zerorpc/channel.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zerorpc/channel.py b/zerorpc/channel.py
index df77bc0..7ef1bb9 100644
--- a/zerorpc/channel.py
+++ b/zerorpc/channel.py
@@ -86,7 +86,7 @@ class ChannelMultiplexer(ChannelBase):
                 queue = self._broadcast_queue

             if queue is None:
-                logger.warning('zerorpc.ChannelMultiplexer,'
+                logger.debug('zerorpc.ChannelMultiplexer,'
                         ' unable to route event: {0}'.format(
                             event.__str__(ignore_args=True)))
             else:
--
2.25.1

